src/v2/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionEntity.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box, color, Flex, Image, Sans } from "@artsy/palette"
+import { Box, Flex, Image, Sans, color } from "@artsy/palette"
```

Comment:
This was from a merge. Not sure why it changed, but it's being auto formatted this way.